src/pages/lessons/[slug].tsx

```diff
@@ -561,9 +577,14 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
                   <RecommendNextStepOverlay lesson={lesson} />
                 </OverlayWrapper>
               )}
+              {playerState.matches('addingNote') && (
+                <OverlayWrapper>
+                  <AddNoteOverlay sendPlayerState={send} />
```

Comment:
```
<AddNoteOverlay onAddNote={() => send('ADD_NOTE')} />
```

Make sense? I like this more because now we aren't sending in all the power of state send method and handling the logical concerns of what happens when we click add note here instead of internally in the sidebar.

Reply:
Yeah, it makes sense 👍

Reply:
Yeah, it makes sense 👍